config/dev.webpack.config.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
     devServer: {
         contentBase: ["./dist"],
         port: 3000,
+        host: "192.168.29.46",
```

Comment:
why are you adding a host IP to this and not just using localhost?

Reply:
Sorry, I was testing on a mobile platform and this caught in, it's supposed to be the loopback IP for my router